server/parsec/components/account.py

```diff
@@ -96,6 +96,23 @@ class BaseAccountComponent:
     def __init__(self, config: BackendConfig):
         self._config = config
 
+    async def get_password_algorithm_or_fake_it(self, email: EmailAddress) -> PasswordAlgorithm:
+        """
+        Note that if email doesn't correspond to a valid account, a made-up
+        algorithm config is returned.
```

Comment:
```suggestion
        Return the password algorithm configuration corresponding to the account
        of the given email address.

        If the email address doesn't correspond to a valid account, a fake (stable) 
        configuration is returned in order to prevent an attacker guessing whether 
        the email address is present in the database or not.
```

Reply:
:+1: done.